tests/test_regression.py

```diff
@@ -485,6 +485,17 @@ def test_30_remove(self):
 
         expected = ['5.150.145.0/24', '109.161.64.0/20']
         self.assertEqual(expected, [n.prefix for n in tree])
+		
+	def test_31_covering_with_delete_error(self):
+        tree = radix.Radix()
+        tree.add('91.187.124.0/24')
+        tree.add('91.187.125.0/24')
+        tree.add('91.187.124.0/23')
+        tree.delete('91.187.124.0/23')
+        tree.add('91.187.124.0/23')
```

Comment:
This last `.add` seems like it would add back the `91.187.124.0/23` making it expected in the check below, correct? I think this should be removed, unless I'm missing something.